### ISSUE TITLE
remove the embedded Azure SDK from Draft once ACR Build is in azure-sdk-for-go

### ISSUE THREAD

**Issue by bacongobbler** [Azure/draft#694]

**Comment by ehotinger** [comment]
@bacongobbler azure-sdk-for-go is published in `/preview` so this item is unblocked now.

Ref: https://github.com/Azure/azure-sdk-for-go/tree/master/services/preview/containerregistry/mgmt/2018-02-01/containerregistry

There should be no major breaking changes from the work you did earlier in #691 but the models may be ever-so-slightly different since I believe you hand-crafted a few properties like `DockerImages`?

**Comment by bacongobbler** [comment]
Yeah, we'll also need to retain some things like [util.go](https://github.com/Azure/draft/blob/master/pkg/azure/containerregistry/util.go) until we contribute that upstream. Thanks for the heads up!
